### DISCUSSION TITLE
Storage size recommendation?

### DISCUSSION THREAD

**Discussion by templeowls21** [blakeblackshear/frigate#13828] [Ask A Question]
Apologies if this is a stupid question. Brand new to Frigate (and cameras in general tbh).

I have 10 IP cameras setup via Frigate and I need an SSD for the recordings, but I'm unsure of what size would be best.

The cameras/recordings are 1080p/10fps. I would ideally like to store the past 3 days of 24/7 recordings for all 10 cameras on a continuous basis. I do not have detection set up. The cameras are really just for 24/7 recording.

Would a 2TB SSD be sufficient? Or do you think I need more? I'm happy to provide more info if I missed anything.

**Comment by hawkeye217** [comment]
Check out this calculator: https://www.jvsg.com/storage-bandwidth-calculator/

2TB is more than sufficient for what you're looking to do.
